docs/dependencies.md

```diff
@@ -125,6 +131,10 @@ Custom syntax to instantiate a package. By default, it's a `new AwesomePackage()
 
 For settings applicable on other platforms, please consult their respective documentation.
 
+#### platforms.android.buildTypes
+
+An array of build variants or flavors which will include the dependency. If the array is empty, your dependency will be included in all build types. If you're working on a helper library, you should set debug as a value, to avoid shipping it as a part of the release process. For more details, see [`build variants`](https://developer.android.com/studio/build/build-variants#dependencies).
```

Comment:
```suggestion
An array of build variants or flavors which will include the dependency. If the array is empty, your dependency will be included in all build types. If you're working on a helper library that should only be included in development, such as a replacement for the React Native development menu, you should set this to `['debug']` to avoid shipping the library in a release build. For more details, see [`build variants`](https://developer.android.com/studio/build/build-variants#dependencies).
```